### ISSUE TITLE
Error processing uu2_gp_hyperparameter_estimation

### ISSUE THREAD

**Issue by csala** [Featuretools/ta1-primitives#19]
The MultiTableFeaturization primitive crashes on the uu2_gp_hyperparameter_estimation dataset when using cross validation with these pipelines: https://gist.github.com/csala/c7408ebedba705e81b1ee24be1de13cc

The command used is:

```
python3 -m d3m runtime -d path/to/datasets evaluate -m pipeline.meta -p pipeline.yml -d kfold_pipeline.yml
```

And the error obtained is:

```
Traceback (most recent call last):
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/runtime.py", line 939, in _do_run_step
    self._run_step(step)
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/runtime.py", line 929, in _run_step
    self._run_primitive(step)
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/runtime.py", line 837, in _run_primitive
    multi_call_result = self._call_primitive_method(primitive.fit_multi_produce, fit_multi_produce_arguments)
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/runtime.py", line 912, in _call_primitive_method
    raise error
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/runtime.py", line 908, in _call_primitive_method
    result = method(**arguments)
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/primitive_interfaces/unsupervised_learning.py", line 49, in fit_multi_produce
    return self._fit_multi_produce(produce_methods=produce_methods, timeout=timeout, iterations=iterations, inputs=inputs)
  File "/home/xals/.virtualenvs/LL1_50words_fail/lib/python3.6/site-packages/d3m/primitive_interfaces/base.py", line 556, in _fit_multi_produce
    fit_result = self.fit(timeout=timeout, iterations=iterations)
  File "/home/xals/.virtualenvs/LL1_50words_fail/src/featuretools-ta1/featuretools_ta1/multi_table.py", line 128, in fit
    es = self._make_entityset(self._inputs)
  File "/home/xals/.virtualenvs/LL1_50words_fail/src/featuretools-ta1/featuretools_ta1/multi_table.py", line 264, in _make_entityset
    es[child_entity_id][child_variable_id]
  File "/home/xals/.virtualenvs/LL1_50words_fail/src/featuretools/featuretools/entityset/entityset.py", line 264, in add_relationship
    child_v, child_e.id, child_dtype))
ValueError: Unable to add relationship because filename in 0 is Pandas dtype category and gpDataFile in learningData is Pandas dtype category.
```

**Comment by csala** [comment]
Also, it's important to note that when executing `fit-score` instead of `evaluate`, it does not crash.

**Comment by thehomebrewnerd** [comment]
This pipeline was fixed in PR #31. Further updates are still needed, but these updates are covered by issue #33
